src/main/native/CCM.c

```diff
@@ -385,17 +385,17 @@ int CCM_encrypt_core(JNIEnv* env, ICC_CTX* ockCtx,
     }
 #ifdef DEBUG_CCM_DATA
     if ( debug ) {
-        gslogMessagePrefix ("DATA_CCM ivNative : ");
-        gslogMessageHex (ivNative, 0, (int) ivLen, 0, 0, NULL);
+        gslogMessagePrefix ("DATA_CCM iv : ");
+        gslogMessageHex ((char*)iv, 0, (int) ivLen, 0, 0, NULL);
```

Comment:
Should we leave a space after the cast (i.e., `(char *) iv` to make it more readable and be consistent with the one right after?

Same comment goes for all the cast spots.

Reply:
Yes a space after the cast is more consistent for this code. Updated to add a space using https://github.com/IBM/OpenJCEPlus/pull/318/commits/b7cf2a703fa95e7b6755333943a19860a02db8cc